frontend/src/components/Contexts/MissionListsContext.tsx

```diff
@@ -31,9 +39,77 @@ const fetchMissions = (params: {
 }): Promise<Mission[]> => {
     return BackendAPICaller.getMissionRuns(params).then((response) => response.content)
 }
-export const useMissions = (refreshInterval: number): MissionsResult => {
+export const useMissions = (): MissionsResult => {
     const [ongoingMissions, setOngoingMissions] = useState<Mission[]>([])
     const [missionQueue, setMissionQueue] = useState<Mission[]>([])
+    const { registerEvent, connectionReady } = useSignalRContext()
+
+    useEffect(() => {
+        if (connectionReady) {
+            registerEvent('mission run created', (username: string, message: string) => {
```

Comment:
Would like to see the events be based on enums or similar rather than pure strings.